<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSongs</class>
 <widget class="QWidget" name="PanelSongs">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>435</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Songs</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QCheckBox" name="checkBoxContent">
       <property name="text">
        <string>Show available content</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0,1">
       <item>
        <widget class="PanelSuperList" name="widgetPanelIDs" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,0">
         <item>
          <widget class="WidgetSuperTree" name="treeViewAvailableContent">
           <property name="minimumSize">
            <size>
             <width>200</width>
             <height>0</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>200</width>
             <height>16777215</height>
            </size>
           </property>
           <attribute name="headerVisible">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,1">
           <item>
            <widget class="QPushButton" name="pushButtonMove">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../ressources.qrc">
               <normaloff>:/icons/Ressources/big_arrow_left.png</normaloff>:/icons/Ressources/big_arrow_left.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonRefresh">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonAdd">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>22</height>
              </size>
             </property>
             <property name="text">
              <string>+</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
         <item>
          <widget class="QGroupBox" name="groupBoxOptions">
           <property name="title">
            <string>Options</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="0" column="0">
             <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0,1">
              <item row="2" column="1" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveValueStart" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="0" column="0">
               <widget class="QLabel" name="labelVolume">
                <property name="text">
                 <string>Volume:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="3">
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>%</string>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QCheckBox" name="checkBoxEnd">
                <property name="text">
                 <string>End</string>
                </property>
               </widget>
              </item>
              <item row="2" column="3">
               <widget class="QLabel" name="labelSeconds1">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>s</string>
                </property>
               </widget>
              </item>
              <item row="3" column="1" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveValueEnd" native="true">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QCheckBox" name="checkBoxStart">
                <property name="text">
                 <string>Start</string>
                </property>
               </widget>
              </item>
              <item row="3" column="3">
               <widget class="QLabel" name="labelSeconds2">
                <property name="enabled">
                 <bool>false</bool>
                </property>
                <property name="text">
                 <string>s</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1" colspan="2">
               <widget class="PanelPrimitiveValue" name="panelPrimitiveValueVolume" native="true">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>27</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>250</width>
                  <height>27</height>
                 </size>
                </property>
               </widget>
              </item>
              <item row="1" column="0" colspan="4">
               <widget class="QSlider" name="horizontalSliderVolume">
                <property name="maximum">
                 <number>100</number>
                </property>
                <property name="singleStep">
                 <number>1</number>
                </property>
                <property name="value">
                 <number>100</number>
                </property>
                <property name="sliderPosition">
                 <number>100</number>
                </property>
                <property name="tracking">
                 <bool>true</bool>
                </property>
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="tickPosition">
                 <enum>QSlider::TicksBelow</enum>
                </property>
                <property name="tickInterval">
                 <number>5</number>
                </property>
               </widget>
              </item>
              <item row="4" column="0" colspan="4">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>218</width>
                  <height>98</height>
                 </size>
                </property>
               </spacer>
              </item>
              <item row="0" column="4">
               <spacer name="horizontalSpacer">
                <property name="orientation">
                 <enum>Qt::Horizontal</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>40</width>
                  <height>20</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="pushButtonPause">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../ressources.qrc">
               <normaloff>:/icons/Ressources/pause_song.png</normaloff>:/icons/Ressources/pause_song.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonStop">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../ressources.qrc">
               <normaloff>:/icons/Ressources/stop_song.png</normaloff>:/icons/Ressources/stop_song.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pushButtonPlay">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../ressources.qrc">
               <normaloff>:/icons/Ressources/play_song.png</normaloff>:/icons/Ressources/play_song.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PanelPrimitiveValue</class>
   <extends>QWidget</extends>
   <header>panelprimitivevalue.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetSuperTree</class>
   <extends>QTreeView</extends>
   <header>widgetsupertree.h</header>
  </customwidget>
  <customwidget>
   <class>PanelSuperList</class>
   <extends>QWidget</extends>
   <header>panelsuperlist.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections/>
</ui>
